File: app/Models/Product.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;
use App\Models\Bid;
class Product extends Model
{
    /** @use HasFactory<\Database\Factories\ProductFactory> */
    use HasFactory, SoftDeletes;

    protected $fillable = ['name', 'description', 'extended_description', 'image_url', 'starter_bid', 'bid_end_date'];
    public function bids()
    {
        return $this->hasMany(Bid::class);
    }
}

File: licitAdminDashboard/licitAdminDashboard/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace licitAdminDashboard.Models
{
    public class Product
    {

        public int Id { get; set; }

        [JsonPropertyName("name")]
        public string Name { get; set; }

        [JsonPropertyName("category")]
        public string Category { get; set; }

        [JsonPropertyName("description")]
        public string Description { get; set; }

        [JsonPropertyName("extended_description")]
        public string ExtendedDescription { get; set; }

        [JsonPropertyName("image_url")]
        public string ImageUrl { get; set; }

        [JsonPropertyName("starter_bid")]
        public decimal StarterBid { get; set; }

        [JsonPropertyName("bid_start_date")]
        public DateTime BidStartDate { get; set; }

        [JsonPropertyName("bid_end_date")]
        public DateTime BidEndDate { get; set; }
    }
}